.github/workflows/wifi_remote__build.yml

```diff
@@ -24,8 +24,9 @@ jobs:
         run: |
           . ${IDF_PATH}/export.sh
           pip install idf-component-manager idf-build-apps --upgrade
+          cp -r ./components/esp_wifi_remote ./components/esp_wifi_remote_base
```

Comment:
Because we generate headers and configs directly into the working tree -- the advantage is that we can directly push the updates.
The reason it worked in the past is described above (section 1, 2nd paragraph) -> git is a bit broken when using idf container and GH ubuntu runner, but instead of addressing `git`, I've decided to simply compare files, since in CI we usually shallow clone, or use some limited history as full history is costly,